index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+	
+	<div class="nav-container">
+		<div class="wrapper">
+			<nav>
+				<div class="logo">Welcome</div>
+				<div class="menu-icon">
+					<a href="#" class="icon-btn">
+						<img src="./images/Icon - Menu.svg" alt="">
+					</a>
+				</div>
+			</nav>
+		</div>
+	</div>
+
+	<div class="background-img">
+		<div>
+	<div class="header-container">
+			<header>
+				<div class="hero-content">
+					<div><h1>Hey there. I'm Mujeeb <br><span>I'm a software developer</span></h1></div>
+					<p>I can help you bulid a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+				</div>
+			</header>
+	</div>
+
+	<div class="social-container">
+		<div class="wrapper">
+			<ul>
+				<div class="box"><li><a href="https://twitter.com/Mujeebu93992980">
+					<img src="./images/twitter.svg" alt="">
+				</a></li></div>
```

Comment:
- [ ] It is semantically incorrect to directly nest a `div` inside a `ul`. Please make sure only `li`s are directly nested inside the `ul`.